README.md

```diff
@@ -78,6 +77,7 @@ You have to replace a few elements in this script:
 - the app package
 - the app activity
 - the bundleId
+(You can use `npx @perf-profiler/profiler getCurrentApp` to display info for the app opened on your phone)
```

Comment:
I moved the comment because I thought it was a very important information and I personally didn't see it, but maybe it's just a preference, let me know I can remove this commit :)